src/extension.ts

```diff
@@ -34,6 +34,11 @@ export async function activate(context: vscode.ExtensionContext) {
   };
 
   const syncWorkspaceFolders = () => {
+    if (extensions.value == null || extensions.value.length == 0) {
+      const msg = "node: test runner:  nodejs-testing.extensions array is empty. Please remove the setting 'nodejs-testing.extensions' or define at least one element.";
+      vscode.window.showErrorMessage(msg);
+      throw new Error(msg);
+    }
```

Comment:
We should return, not throw an error.

```suggestion
    if (!extensions.value?.length) {
      const msg = "nodejs-testing.extensions array is empty. Please remove the setting 'nodejs-testing.extensions' or define at least one element.";
      vscode.window.showErrorMessage(msg);
      return;
    }
```

Reply:
Done,

Reply:
(with !extensions.value?.length==0)